src/main/java/com/solvd/movie/kafka/parser/XmlParser.java

```diff
@@ -0,0 +1,23 @@
+package com.solvd.movie.kafka.parser;
+
+import com.jcabi.xml.XML;
+import com.jcabi.xml.XMLDocument;
+import lombok.SneakyThrows;
+
+import java.util.Objects;
+
+public abstract class XmlParser {
```

Comment:
utility class = final class + private ctor

In your case, I can still construct your ```XmlParser``` using simple inheritance

Reply:
Fixed